### PR TITLE
Issue a

### PR DESCRIPTION
**Pull request by bollupallavi**

### CODE REVIEW

**Review by charankumarpalla — CHANGES_REQUESTED** [review]
pls change

File: readme.md [hunk 1]
```diff
@@ -1 +1 @@
-pallavi
\ No newline at end of file
+day zone
```
Comment (charankumarpalla) [inline readme.md]: change it to new day zone

Comment (bollupallavi) [reply readme.md]: changes made

**Review by charankumarpalla — APPROVED** [review]
looks good to me

**Pull request merged**